### PR TITLE
CDAP-13936 remove outdated ui documentation

### PR DESCRIPTION
**Pull request by albertshau**

### CODE REVIEW

**Comment by albertshau** [comment]
https://builds.cask.co/browse/CDAP-DQB424

**Review by elfenheart — DISMISSED** [review]
lgtm

File: cdap-docs/admin-manual/source/operations/cdap-ui.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ CDAP UI
 =======
 
 The **CDAP UI** is available for deploying, querying and managing the Cask Data
-Application Platform in all modes of CDAP except as an 
+Application Platform in all modes of CDAP except as an
```
Comment (bdmogal) [inline cdap-docs/admin-manual/source/operations/cdap-ui.rst]: This statement seems wrong.. the UI doesn't really help to deploy CDAP. Shall we replace with:

CDAP contains a web UI for deploying and managing applications and datasets. In addition, it also contains UIs for the data pipeline and data preparation applications.

File: cdap-docs/developer-manual/source/getting-started/building-apps.rst [hunk 2]
```diff
@@ -125,7 +125,7 @@ Once CDAP is started, you can deploy an application using an example JAR by any
 
     examples/|example-dir|/target/|example|-|release|.jar
 
-- In the :ref:`new beta CDAP UI <cdap-ui-beta>`,  use the green "plus" button |---| found on either
+- In the CDAP UI, use the green "plus" button |---| found on either
   the |home-page|_ or |management-page|_ pages |---| to bring up the *Hub/Resource
```
Comment (bdmogal) [inline cdap-docs/developer-manual/source/getting-started/building-apps.rst]: oh gosh this page is outdated too. @albertshau let me know if you want me to push a commit.

Comment (albertshau) [reply cdap-docs/developer-manual/source/getting-started/building-apps.rst]: go for it

**Comment by bdmogal** [comment]
@albertshau @elfenheart pushed a commit. please review. running build - https://builds.cask.co/browse/CDAP-DQB424-3

**Review by albertshau — COMMENTED** [review]
lgtm

**Pull request merged**